src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/Create/cart.html.twig

```diff
@@ -218,6 +218,21 @@
   </tr>
 </script>
 
+<script id="products-table-gift-row-template" type="text/template">
```

Comment:
Why not `<template>`? Should we support IE11?

Reply:
i guess we should :smile:

Reply:
I hope this kind of template is working everywhere :)

Reply:
If im right you offered to use this instead of text/html that i was using before in some other pr :smile:

Reply:
You're maybe right :sweat_smile: